### ISSUE TITLE
Duplicate "Edit locations at this facility" links in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10159]
## Description
On the edit screens in the CMS for certain content types, there are sections for each facility. Each of these sections has a link that says "Edit locations at this facility (opens in a new window)" and this can cause duplicate links on the same page when there are multiple facilities.  Ideally each link should have a distinct, unique text to indicate to the user where the link would direct them - "Edit locations at Jamaica Plain VA Medical Center (opens in a new tab)"

This is occurring on these content types:
- VAMC System Billing and Insurance
- VAMC System Medical Records Office
- VAMC System Register for Care

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184159513-53dc4dc9-324c-4110-8bf0-0de14eaa8d20.png)

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

## QA and testing
- [ ] Functional testing:
- [ ] Drupal Code review:
- [ ] Accessibility review: @laflannery 

## Acceptance Criteria
- [ ] Links are unique within the page

**Comment by davidmpickett** [comment]
@laflannery I have a fix in PR to this issue on:
- VAMC System Billing and Insurance
- VAMC System Medical Records Office
- VAMC System Register for Care

I don't see this issue on VAMC System Locations List.
![Image](https://github.com/user-attachments/assets/328d1fec-bb08-44de-8880-febb7538d964)

Can you confirm I'm not missing something?

**Comment by davidmpickett** [comment]
**End of Sprint update**

- This wasn't in Sprint 15, or even in Ready
- I picked up while working on #19512 since it deals with a similar issue and is in my wheelhouse
- Waiting on functional testing

**Comment by davidmpickett** [comment]
**End of Sprint 16 update**
- No progress this sprint
- Asked @FranECross in [Slack](https://dsva.slack.com/archives/C03LFSPGV16/p1731452525077489?thread_ts=1731445457.727559&cid=C03LFSPGV16) if we could assign Functional Testing and Code Review roles tomorrow

**Comment by davidmpickett** [comment]
@laflannery I forgot to have you look at this before it merged. This is live on Prod, but wanted to give you a chance to look before I close

- https://prod.cms.va.gov/maryland-health-care/billing-and-insurance
- https://prod.cms.va.gov/maryland-health-care/register-for-care
- https://prod.cms.va.gov/maryland-health-care/medical-records-office

![Image](https://github.com/user-attachments/assets/68b6e40f-1f98-49c7-8fec-3f5e046fdff8)
![Image](https://github.com/user-attachments/assets/5ecbab22-93ce-4514-b8c1-f6aeff77b464)
![Image](https://github.com/user-attachments/assets/94b16aaa-4384-45e2-b494-12a8c241af26)

**Comment by laflannery** [comment]
Looks good!
